tests/it/it_scargo_commands_flow.py

```diff
@@ -322,6 +322,18 @@ def test_cli_build(self, test_state: ActiveTestState) -> None:
         for file in test_state.get_build_result_files_paths():
             assert file.is_file(), f"Expected file: {file} not exist"
 
+    @pytest.mark.skip("TODO when conan version be updated to >= 2.0.0")
```

Comment:
this change should not be here

Reply:
I didn't add this code, but I can delete it.